app/controllers/bookmarks_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
 
```

Comment:
Check whitespace formatting for this file, it looks like there's a mix of tabs and soft spaces. Sublime might need configuring to use soft spaces instead of tabs.